model/task_queue.go

```diff
@@ -19,9 +19,10 @@ import (
 
 const (
 	TaskQueuesCollection = "task_queues"
-	useModernDequeueOp   = false
 )
 
+var useModernDequeueOp = true
```

Comment:
let's avoid mutable global state, just change it in the constructor.

Reply:
This has the nice feature of allowing calling dequeue without having the task queue object https://github.com/evergreen-ci/evergreen/blob/c65f469b1bd7a7162d6db3c86f87b413af1a7f7a/model/task_queue.go#L223.

Reply:
Is that actually nice?